types/prosemirror-tables/index.d.ts

```diff
@@ -5,16 +5,15 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 import { EditorState, Plugin, SelectionRange, Transaction } from 'prosemirror-state';
-import { Node, NodeSpec, Slice, ResolvedPos } from 'prosemirror-model';
-import * as dom from './dom';
+import { Node as ProsemirrorNode, NodeSpec, Slice, ResolvedPos } from 'prosemirror-model';
 
 export interface TableNodesOptions {
   tableGroup?: string;
   cellContent: string;
   cellAttributes: { [key: string]: CellAttributes };
 }
 
-export type getFromDOM = (dom: dom.DOMElement) => any;
+export type getFromDOM = (dom: Element) => any;
```

Comment:
This is an invalid change unfortunately, which just broke our stuff 👎 
I'll try to get it reverted...